src/homework_6/cloning/ElectricCar.java

```diff
@@ -0,0 +1,29 @@
+package homework_6.cloning;
+
+public class ElectricCar extends Car {
```

Comment:
read about some of the way to realize deep cloning
https://www.baeldung.com/java-deep-copy